Blog post - Code Reviews

jakehall-IW opened the pull request:
New blog post about the value of code reviews.

Comments and criticism as always valued!

I've posted it on my blog, but I've included the actual post here for preservation and to make it easier to review 😄

ivorc commented on _posts/2017-12-21-code-reviews.md:
```diff
@@ -0,0 +1,71 @@
+* follows agreed best practice, and coding standards
+* code is understandable, maintainable and clean
+* all automated tests are written, and have the necessary coverage
+* the acceptance criteria on the ticket has been met
```
Should this be "have" instead of "has", since criteria is plural?

jakehall-IW replied:
Changed!

ivorc commented on _posts/2017-12-21-code-reviews.md:
```diff
@@ -0,0 +1,71 @@
+
+# How do we do code reviews? #
+
+In order to gain the most from our code reviews, we try and stick the following: 
```
Should this be "stick *to* the following..."?

jakehall-IW replied:
Changed!

ivorc commented on _posts/2017-12-21-code-reviews.md:
```diff
@@ -0,0 +1,71 @@
+
+## Prevent Issues ##
+
+Code reviews are an additional gate, and more pairs of eyes on a change. This can increase the likelihood of picking up issues before they out of the development environment where fixing is much quicker and cheaper. At this point the developer is unlikely to have context switched too, so can make the change relatively easily.
```
Should this be "...before they *are* out of the development environment..."?

jakehall-IW replied:
well spotted - added in, thanks 😁

jakehall-IW commented:
@neildunlop thanks - I've shamelessly copied your summary as I couldn't put it better myself, and added a link to LGTM.

@PaulLucas-IW thanks for the suggestion on a new section, I've added it in. Also corrected the Scouts Honour bit.
